orders/models.py

```diff
@@ -71,3 +73,38 @@ def cancel_order(self):
 
     def get_order_total_price(self):
         return self.price_per_day * (self.end_date - self.start_date).days
+
+    @staticmethod
+    def get_capacity_of_daycare_in_dates_range(daycare_id, start_date, end_date):
```

Comment:
There is a LOT of logic here. how does it not have a test?

Reply:
I have mentioned in the commit message "get_capacity_of_daycare_in_dates_range" function took from https://github.com/redhat-beyond/Hotails/pull/95 .
waiting for approval." , and in the PR description that this PR depends on #95.
After #95 will merge test implementation should be there as well .